docs/examples/test/test_retry.py

```diff
@@ -50,6 +50,19 @@ def test_rpc(self, container_factory, rabbit_config):
             res = service_rpc.method(timestamp.isoformat())
         assert arrow.get(re.match("Time is (.+)", res).group(1)) >= timestamp
 
+    def test_decorated_rpc(self, container_factory, rabbit_config):
```

Comment:
Feels a little out of place here. Perhaps move this to its own class called `TestDecorator`?